src/content/en/updates/2016/10/avoid-not-secure-warn.md

```diff
@@ -0,0 +1,67 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: "Chrome will soon mark non-secure pages containing password and credit card input fields as Not Secure in the URL bar."
+
+{# wf_updated_on: 2016-10-24 #}
+{# wf_published_on: 2016-10-24 #}
+{# wf_featured_image: /web/updates/images/generic/images/warning.png #}
+{# wf_featured_snippet: Chrome will soon mark non-secure pages containing password and credit card input fields as Not Secure in the URL bar. This document is intended to aid Web Developers in updating their sites to avoid this warning. #}
+
+# Avoiding the Not Secure Warning in Chrome {: .page-title }
+
```

Comment:
As I said earlier, I've sent him a note to add himself. I didn't want to hold this up. I can add it later.